build.gradle

```diff
@@ -223,6 +223,17 @@ task pushDockerImage {
     }
 }
 
+task testInitNakadi(type: Exec) {
```

Comment:
Here is an improvement https://github.com/zalando/nakadi/pull/395/commits/9a1c210aa5d71aa0b2855c685980958d8f5ed4da

I managed to use a different profile, but in order to do so I still need to have this environment variable set.

The advantage of the latest approach is that it's now easy to define any new values by just updating a configuration file. There will be no further changes to the gradle file.